### ISSUE TITLE
Bring xlsx up-to-date to prevent npm audit vulnerability report

### ISSUE THREAD

**Issue by chazzmoney** [gamemaker1/office-text-extractor#8]
### Description

npm audit results in

```
% npm audit
# npm audit report

xlsx  *
Severity: high
Prototype Pollution in sheetJS - https://github.com/advisories/GHSA-4r6h-8v6p-xvw6
No fix available
node_modules/xlsx
  office-text-extractor  *
  Depends on vulnerable versions of xlsx
  node_modules/office-text-extractor

2 high severity vulnerabilities

Some issues need review, and may require choosing
a different dependency.
```

As per [https://docs.sheetjs.com/docs/getting-started/installation/nodejs#legacy-endpoints](https://docs.sheetjs.com/docs/getting-started/installation/nodejs#legacy-endpoints), you can update the xlsx dependency in your project via:

```
pnpm rm xlsx
pnpm install https://cdn.sheetjs.com/xlsx-0.20.0/xlsx-0.20.0.tgz
```

This will remove the vulnerability error and bring xlsx up to date with the current published version.

### Library version

^3.0.1

### Node version

v20.3.1

**Comment by gamemaker1** [comment]
Hi, thanks for reporting this! I've released v3.0.2 with the updated dependency.

**Comment by chazzmoney** [comment]
I just wanted to say thank you and complement you for responding so very quickly. Well done; your hard work is greatly appreciated!
